bridge/endpointActions/GetProbingInfo.cpp

```diff
@@ -0,0 +1,45 @@
+#include "ApiActions.h"
+#include "api/ConferenceEndpoint.h"
+#include "bridge/Mixer.h"
+#include "bridge/MixerManager.h"
+#include "bridge/RequestLogger.h"
+#include "nlohmann/json.hpp"
+#include "transport/ProbeServer.h"
+
+namespace bridge
+{
+httpd::Response getProbingInfo(ActionContext* context,
+    RequestLogger& requestLogger,
+    const httpd::Request&,
+    const utils::StringTokenizer::Token&)
+{
+    nlohmann::json responseBodyJson;
+    nlohmann::json iceJson;
+
+    auto& probeServer = context->probeServer;
+
+    const auto credentials = probeServer.getProbeCredentials();
+
+    iceJson["ufrag"] = credentials.first;
+    iceJson["pwd"] = credentials.second;
+
+    nlohmann::json candidatesJson = nlohmann::json::array();
+
+    for (auto candidate : probeServer.getProbeCandidates())
+    {
+        nlohmann::json candidateJson;
+        candidateJson["protocol"] = candidate.transport == ice::TransportType::TCP ? "tcp" : "udp";
+        candidateJson["ip"] = candidate.address.ipToString();
+        candidateJson["port"] = candidate.address.getPort();
+        candidatesJson.push_back(candidateJson);
+    }
+
+    iceJson["candidates"] = candidatesJson;
+    responseBodyJson["ice"] = iceJson;
+
+    httpd::Response response(httpd::StatusCode::OK, responseBodyJson.dump(4));
+    response._headers["Content-type"] = "text/json";
```

Comment:
I think we should stick to regular letters.

Reply:
Now we use "Content-type" in all the endpoints. Should I change all of them now?

Reply:
I mistook it for part of json. It is part of HTTP response so that is fine.